CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 0.12.18
 
+* Add an `addTearDown()` function, which allows tests to register additional
+  tear-down callbacks as they're running.
+
```

Comment:
Are you intentionally wanting `<p>` tags inside the `<li>`?

I wouldn't normally expect an extra blank line between bullet points